3.processing_features/scripts/feature_select_sc.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Perform feature selection on normalized merged single cells for each plate
+
+# ## Import libraries
+
+# In[1]:
+
+
+import sys
+import pathlib
+import os
+
+import pandas as pd
+from pycytominer import feature_select
+from pycytominer.cyto_utils import output
+
+sys.path.append("../utils")
+import extraction_utils as sc_utils
+
+# ## Set paths and variables
+
+# In[2]:
+
+
+# output directory for feature selected data
+output_dir = pathlib.Path("./data/feature_selected_data")
+# if directory if doesn't exist, will not raise error if it already exists
+os.makedirs(output_dir, exist_ok=True)
+
+# dictionary with each run for the cell type
+plate_info_dictionary = {
```

Comment:
Optional suggestion: Could this dictionary be passed as a result between steps for use in individual notebooks? For example:
1. Notebook1 could take `plate_info_dictionary` as a dictionary to run pipelines and be updated after pipelines run.
2. The dictionary could be exported as `JSON`, `YAML`, or `TOML` to file (on completing work in Notebook1).
3. Notebook2 (most similar to this notebook probably) could read the `JSON`, `YAML`, or `TOML` file as a dictionary including the values needed to run below.

Making this change might require upfront work but might mean less rewriting / rework over time.

Reply:
I have now implemented this and it works!

Took a lot of banging my head haha